module/documents/item.mjs

```diff
@@ -1909,7 +1909,12 @@ export default class Item5e extends Item {
   _onCreateOwnedWeapon(data, isNPC) {
 
     // NPCs automatically equip items and are proficient with them
-    if ( isNPC ) return {"system.equipped": true, "system.proficient": true};
+    if ( isNPC ) {
+      const updates = {};
+      if ( foundry.utils.getProperty(data, "system.equipped") === undefined ) updates["system.equipped"] = true;
+      if ( foundry.utils.getProperty(data, "system.proficient") === undefined ) updates["system.proficient"] = true;
```

Comment:
Perhaps this might be a bit cleaner as:
```suggestion
      if ( foundry.utils.hasProperty(data, "system.equipped") ) updates["system.equipped"] = true;
      if ( foundry.utils.hasProperty(data, "system.proficient") ) updates["system.proficient"] = true;
```

Reply:
Sure would; cheers. Amended my commit